src/app/frontend/common/components/responsivetable/responsivetable_directive.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Returns directive definition for responsive table.
+ *
+ * @return {!angular.Directive}
+ */
+export default function responsiveTableDirective() {
+  return {
+    restrict: 'A',
+    compile: function(element, attrs) {
```

Comment:
Hey, you see what this code does? It just adds a class and sets data-title attributes. I dont think we need this at all. Wouldn't CSS just suffice?